### PR TITLE
Add docs roon services

### PR DESCRIPTION
**Pull request by pavoni**
## Proposed change

I've proposed a PR to add some platform services to the Roon media player, this is the matching documentation.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/42133
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/roon.markdown [hunk 1]
```diff
@@ -28,3 +28,31 @@ You need the Hostname or IP address of the machine that runs your Roon Core. Thi
 3. Home Assistant will then contact your Roon Core and ask to be authorized. You will need to enable this extension in the Room Application. Go to **Settings** and then **Extensions**, there you will see an entry for Home Assistant with a button next to it. Click **Enable**.
 4. Roon core will then provide Home Assistant with the details of your media players.
 5. In Home Assistant you can then pick an area for each of your music players, and add them to Home Assistant.
+
+### Service `join`
```
Comment (frenck) [inline source/_integrations/roon.markdown]:
```suggestion
## Services

### Service `roon.join`
```

File: source/_integrations/roon.markdown [hunk 2]
```diff
@@ -28,3 +28,31 @@ You need the Hostname or IP address of the machine that runs your Roon Core. Thi
...
+
+Group players together.
+
+| Service data attribute | Optional | Description |
+| ---------------------- | -------- | ----------- |
+| `entity_id` | no | id of the player that will be the master of the group.
```
Comment (frenck) [inline source/_integrations/roon.markdown]: This service is registered using `async_register_entity_service`, which means the `entity_id` is optional (as a group of entities in an area can be targeted alternatively).

```suggestion
| `entity_id` | yes | ID of the player that will be the master of the group.
```

File: source/_integrations/roon.markdown [hunk 3]
```diff
@@ -28,3 +28,31 @@ You need the Hostname or IP address of the machine that runs your Roon Core. Thi
...
+| `join_ids` | no | id(s) of the players that will join the master.
+
+### Service `unjoin`
```
Comment (frenck) [inline source/_integrations/roon.markdown]:
```suggestion
### Service `roon.unjoin`
```

File: source/_integrations/roon.markdown [hunk 4]
```diff
@@ -28,3 +28,31 @@ You need the Hostname or IP address of the machine that runs your Roon Core. Thi
...
+
+Remove players from a group.
+
+| Service data attribute | Optional | Description |
+| ---------------------- | -------- | ----------- |
+| `entity_id` | no | id of the player that is the master of the group.
```
Comment (frenck) [inline source/_integrations/roon.markdown]: This service is registered using `async_register_entity_service`, which means the `entity_id` is optional (as a group of entities in an area can be targeted alternatively).

```suggestion
| `entity_id` | yes | ID of the player that is the master of the group.
```

File: source/_integrations/roon.markdown [hunk 5]
```diff
@@ -28,3 +28,31 @@ You need the Hostname or IP address of the machine that runs your Roon Core. Thi
...
+| `unlink_name` | yes | id(s) of the players that will be unjoined from the group. If not specified, all players will be unjoined from the master.
+
+
+### Service `transfer`
```
Comment (frenck) [inline source/_integrations/roon.markdown]:
```suggestion
### Service `roon.transfer`
```

File: source/_integrations/roon.markdown [hunk 6]
```diff
@@ -28,3 +28,31 @@ You need the Hostname or IP address of the machine that runs your Roon Core. Thi
...
+
+Transfer playback from one player to another.
+
+| Service data attribute | Optional | Description |
+| ---------------------- | -------- | ----------- |
+| `entity_id` | no | id of the source player.
```
Comment (frenck) [inline source/_integrations/roon.markdown]: This service is registered using `async_register_entity_service`, which means the `entity_id` is optional (as a group of entities in an area can be targeted alternatively).

```suggestion
| `entity_id` | yes | id of the source player.
```

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks for the PR @pavoni!

I've reviewed it and left some comments. Could you take a look?

Additionally, I noticed that the `### Configuration` heading is a level 3 heading, while it should have been level 2. Maybe adjust that while you are at it?

**Comment by pavoni** [comment]
Thanks @frenck - have updated with your suggestions.

I've added your `### Configuration` suggestion to my other PR against `current` https://github.com/home-assistant/home-assistant.io/pull/15339

I'm very happy to refactor the main body of that PR if you can let me know how you'd prefer to approach documenting the limitations of that service call for the roon player.

And thanks for taking the time - can imagine you're very busy with the combination of `hacktoberfest` and the `0.117` release.

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Comment by pavoni** [comment]
Just fixed the conflicts with the changes to `current`

**Review by frenck — APPROVED** [review]

**Pull request merged**